enhancements/proxy/global-cluster-egress-proxy.md

```diff
@@ -0,0 +1,285 @@
+---
+title: global-cluster-egress-proxy
+authors:
+  - "@bparees"
+  - "@danehans"
+reviewers:
+  - "@deads"
+  - "@derekwaynecarr"
+  - "@knobunc"
+  - "@eparis"
+approvers:
+  - "@derekwaynecarr"
+  - "@eparis"
+  - "@knobunc"
+creation-date: 2019-10-04
+last-updated: 2019-10-04
+status: implemented
+see-also:
+  - "https://github.com/openshift/enhancements/pull/22"  
+---
+
+# Global Cluster Egress Proxy
+
+## Release Signoff Checklist
+
+- [ x ] Enhancement is `implementable`
+- [ x ] Design details are appropriately documented from clear requirements
+- [ x ] Test plan is defined
+- [ x ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Open Questions [optional]
+
+## Summary
+
+Various OpenShift infrastructure components have a need to make requests
+to services the reside off-cluster.  Customers may require traffic that
+goes outside their network to go through a proxy.  Therefore OpenShift
+infrastructure components that need to make requests to external services
+may need to go through a proxy.
+
+The goal of this proposal is to define a mechanism for publishing and consuming
+configuration information related to the proxy to be used when making external
+requests.  This includes information about the proxy hostname as well as any
```

Comment:
The hostname of the proxy or the proxied hostname(s), ip addrs, etc.? If the later, it may be helpful to reword to avoid confusion.

Reply:
i meant the former.....the proxy config literally contains the hostname of the proxy server.

Reply:
If it's the former, should the wording expand from the proxy hostname to the proxy connection information. For example, the proxy connection information includes a url scheme, a hostname/ip of the proxy, port number, etc..

Reply:
sure, that is clearer..will update.